OrbitQt/orbitsamplingreport.cpp

```diff
@@ -92,11 +93,12 @@ void OrbitSamplingReport::on_PreviousCallstackButton_clicked() {
 void OrbitSamplingReport::OnCurrentThreadTabChanged(int current_tab_index) {
   auto treeView = m_OrbitDataViews[current_tab_index];
   QModelIndexList index_list = treeView->GetTreeView()->selectionModel()->selectedIndexes();
-  std::optional<int> row = std::nullopt;
-  if (!index_list.isEmpty() && index_list.front().isValid()) {
-    row = index_list.front().row();
+  std::set<int> row_set;
+  for (QModelIndex& index : index_list) {
+    row_set.insert(index.row());
   }
```

Comment:
Thanks for pointing out this. As we discussed offline, this two-step process is used to avoid the duplicated rows rather than the duplicated indices. And I totally agree `OrbitTableModel::OnMultiRowsSelected` is a better place to handle this duplicates problem.

Will move it to  `OrbitTableModel::OnMultiRowsSelected` !